### PR TITLE
[CDAP-17442] Fix post-run notifications for deployed pipelines

### PR DESCRIPTION
**Pull request by njbriggs**
Jira: https://cdap.atlassian.net/browse/CDAP-17442

This fixes an issue with popovers from the material-ui upgrade. It also adds a Cypress test to cover this flow.

This will need to be cherry-picked for 6.3.

### CODE REVIEW

**Comment by njbriggs** [comment]
E2E: https://builds.cask.co/browse/IT-UPD2206

File: cdap-ui/app/cdap/components/PostRunActions/PostRunActions.scss [hunk 1]
```diff
@@ -85,30 +87,95 @@ $fourth-column-width: 53px;
...
-              .icon-asterisk {
-                top: 5px;
-              }
-            }
-          }
-        }
-        .modal-footer {
-          padding-bottom: 0;
-          border-top: 1px solid #cccccc;
+.post-run-actions-modal.cdap-modal.modal-dialog {
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/PostRunActions/PostRunActions.scss:90]: Seems like a lot of styles are duplicated here from older sass framework. Can we make them css-in-js if we are converting everything? We can have a modal style (not necessarily a modal component) and use them in the post run actions modal?

File: cdap-ui/app/cdap/components/PostRunActions/PostRunActions.scss [hunk 2]
```diff
@@ -85,30 +87,95 @@ $fourth-column-width: 53px;
...
+  overflow: hidden;
+  height: 100%;
+  width: 100%;
+  margin-top: 0;
+  margin: 0;
+
+  .modal-header {
+    background-color: #ebecf1;
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/PostRunActions/PostRunActions.scss:98]: nit: Use color variable?

**Review by ajainarayanan — COMMENTED** [review]
@njbriggs Had some comments.

I had originally thought it was a z-index issue which required a simpler fix. If you are planning to extract out the style might as well do it as css-in-js module.

**Comment by njbriggs** [comment]
@ajainarayanan I didn't look at the z-index because previously, combing the old modal and material-ui popover gave problems with click propagation. I'll take a quick look though and see if that works.

**Comment by njbriggs** [comment]
Looks like changing the z-index works in this case. I'll create a new PR for a cleaner history and bring over the new E2E test.

**Comment by njbriggs** [comment]
Replaced by #12933

**Pull request closed without merging**